packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -73,7 +73,6 @@ class DriveCommand extends RunCommandBase {
     addMultidexOption();
     argParser
       ..addFlag('keep-app-running',
-        defaultsTo: null,
```

Comment:
hmm, interesting. So we DID hit the fallback in `boolArgDeprecated`. In any case, having the flag default to `false` is better, so good cleanup.

Reply:
Yeah, this was the only such occurrence in the whole codebase, kinda bizarre.